vpn/src/main/java/com/duckduckgo/mobile/android/vpn/processor/tcp/TCBCloser.kt

```diff
@@ -42,7 +42,7 @@ class TCBCloser @Inject constructor(private val socketWriter: TcpSocketWriter) {
         isFIN: Boolean,
     ) {
         val buffer = ByteBufferPool.acquire()
-        val tcb = requireNotNull(connectionParams.tcb())
+        val tcb = connectionParams.tcb() ?: return
```

Comment:
@anikiki I incidentally relaxed this `requireNotNull` calls to avoid unnecessary VPN restarts.
Realised of this while testing. When we turn OFF and back ON the VPB sometimes we'd get `null` TCBs because unfortunately the structure that holds them is a `static` one. Crashing is fine because we'd detect that and bring the VPN back up, but continuing is also fine and prob more desirable for now to avoid "too many" unnecessary restarts.

Added a bit of explanation to the PR description